test/gui/tst_syncing/test.feature

```diff
@@ -489,7 +489,6 @@ Feature: Syncing files
         And the folder "test-folder/sub-folder2" should exist on the file system
         And the folder "test-folder/sub-folder1" should not exist on the file system
 
-    @issue-11814 @skip
     Scenario: remove folder sync connection
```

Comment:
Please, keep the issue tag, it will serve as docs for later (if the crash or test fails again)
```suggestion
    @issue-11814
    Scenario: remove folder sync connection
```